twisted/plugins/multichain_crawler_plugin.py

```diff
@@ -0,0 +1,89 @@
+import os
+import signal
+import logging.config
+
+from twisted.application.service import IServiceMaker, MultiService
+from twisted.internet import reactor
+from twisted.plugin import IPlugin
+from twisted.python import usage
+
+from zope.interface import implements
+
+from Tribler.community.multichain.community import MultiChainCommunityCrawler
+from Tribler.dispersy.crypto import ECCrypto
+from Tribler.dispersy.dispersy import Dispersy
+from Tribler.dispersy.endpoint import StandaloneEndpoint
+
+if os.environ.get('HOME'):
+    tribler_dir = os.path.join(unicode(os.environ.get('HOME')), u'.multichain')
+else:
+    tribler_dir = u'.multichain'
+
+
+class Options(usage.Options):
+    optParameters = [
+        ["statedir", "s", tribler_dir, "Use an alternate statedir"                                    , unicode],
+        ["ip"      , "i", "0.0.0.0" ,  "Dispersy uses this ip"                                        , str],
+        ["port"    , "p", 6421      ,  "Dispersy uses this UDP port"                                  , int],
+    ]
+
+
+class MultichainCrawlerServiceMaker(object):
+    implements(IServiceMaker, IPlugin)
+    tapname = "multichain_crawler"
+    description = "A MultichainCommunity crawler"
+    options = Options
+
+    def makeService(self, options):
+        tracker_service = MultiService()
+        tracker_service.setName("Multichain Crawler")
+
+        def run():
+            # setup
```

Comment:
I think this comment could either be removed or rewritten to make it more clear what's happening (i.e. `setup the logger system because ...`)